rpp/browser.py

```diff
@@ -57,6 +58,62 @@ def get_path(self) -> str:
                 self.log.warning("Browser path not found at %s", path)
         return path
 
+    def is_microsoft_store_app(self) -> bool:
```

Comment:
Instead of this method, the property and property.setter for **microsoft_store** should be set.